.all-contributorsrc

```diff
@@ -79,7 +79,17 @@
         "code",
         "ideas"
       ]
-    }
+    },
+    {
+      "login": "edwardchalstrey1",
+      "name": "edwardchalstrey1",
+      "avatar_url": "https://www.gravatar.com/avatar/86760965588048d266a6bf9f50f18a16",
+      "profile": "https://github.com/edwardchalstrey1",
+      "contributions": [
+        "code",
+        "doc"
+      ]
+    },
```

Comment:
trailing comma is added

Unfortunately the JSON specification does not allow a trailing comma. 

Can we ignore this bad design here?